src/coreComponents/constitutive/relativePermeability/TableRelativePermeabilityHysteresis.cpp

```diff
@@ -166,7 +166,10 @@ void TableRelativePermeabilityHysteresis::postInputInitialization()
 
   using IPT = TableRelativePermeabilityHysteresis::ImbibitionPhasePairPhaseType;
 
+
```

Comment:
Maybe assign here member variable 'numDir' and check that it is consistent across all tables

Reply:
Yes, that could work.  Also need to add capability to use whichever input string is present out of optional inputs.